backend/src/api/bitcoin/bitcoin-api.ts

```diff
@@ -294,15 +295,28 @@ class BitcoinApi implements AbstractBitcoinApi {
     const b: string[] = [];
     a.forEach((chunk) => {
       if (chunk.substr(0, 3) === 'OP_') {
-        chunk = chunk.replace(/^OP_(\d+)/, 'OP_PUSHNUM_$1');
+        chunk = chunk.replace(/^OP_(\d+)$/, 'OP_PUSHNUM_$1');
         chunk = chunk.replace('OP_CHECKSEQUENCEVERIFY', 'OP_CSV');
+        chunk = chunk.replace('OP_CHECKLOCKTIMEVERIFY', 'OP_CLTV');
         b.push(chunk);
       } else {
         chunk = chunk.replace('[ALL]', '01');
         if (chunk === '0') {
           b.push('OP_0');
+        } else if (chunk.match(/^[^0]\d*$/)) {
+          const chunkInt = parseInt(chunk, 10);
+          if (chunkInt < 0) {
+            b.push('OP_PUSHNUM_NEG' + -chunkInt);
+          } else {
+            b.push('OP_PUSHNUM_' + chunk);
+          }
         } else {
-          b.push('OP_PUSHBYTES_' + Math.round(chunk.length / 2) + ' ' + chunk);
+          const dataLength = Math.round(chunk.length / 2);
+          if (dataLength > 75) {
```

Comment:
If the chunk length is larger than 255, OP_PUSHDATA2 is used (and when larger than 65535 OP_PUSHDATA4 but it is never used because bitcoin doesnt allow pushes larger than 520 bytes)